spark-connector/spark-common/src/main/java/com/datastrato/gravitino/spark/connector/iceberg/SparkIcebergTable.java

```diff
@@ -37,7 +36,7 @@ public SparkIcebergTable(
       PropertiesConverter propertiesConverter,
       SparkTransformConverter sparkTransformConverter,
       SparkTypeConverter sparkTypeConverter) {
-    super(sparkTable.table(), !isCacheEnabled(sparkCatalog));
+    super(sparkTable.table(), true);
```

Comment:
Can you please write like  this`super(sparkTable.table(), true /** refreshEagerly */)` ?

Reply:
done